data_structures/binary_tree/binary_tree_traversals.py

```diff
@@ -0,0 +1,149 @@
+class Node:
+    """
+    A Node has data variable and pointers toits left and right nodes.
+    """
+    def __init__(self,data):
+        self.left=None
+        self.right=None
+        self.data=data
+
+def preOrder(root):

```

Comment:
```suggestion
def preorder(root):  
```
or `pre_order` but no uppercase characters in Python function names as discussed in CONTRIBUTING.md.

Reply:
ok, Thanks for your suggestions.